cadquery/cq.py

```diff
@@ -1681,8 +1684,33 @@ def spline(
 
         :param listOfXYTuple: points to interpolate through
         :type listOfXYTuple: list of 2-tuple
-        :param tangents: tuple of Vectors specifying start and finish tangent
+        :param tangents: vectors specifying the direction of the tangent to the
+            curve at each of the specified interpolation points.
+                If only 2 tangents are given, they will be used as the initial and
+            final tangent.
+                If some tangents are not specified (i.e., are None), no tangent
+            constraint will be applied to the corresponding interpolation point.
+                The spline will be C2 continuous at the interpolation points where
+            no tangent constraint is specified, and C1 continuous at the points
+            where a tangent constraint is specified.
         :param periodic: creation of periodic curves
+        :param parameters: the value of the parameter at each interpolation point.
+            (The intepolated curve is represented as a vector-valued function of a
+            scalar parameter.)
+                If periodic == True, then len(parameters) must be
+            len(intepolation points) + 1, otherwise len(parameters) must be equal to
+            len(interpolation points).
+        :param scale: whether to scale the specified tangent vectors before
+            interpolating.
+                Each tangent is scaled, so it's length is equal to the derivative of
+            the Lagrange interpolated curve.
+                I.e., set this to True, if you want to use only the direction of
+            the tangent vectors specified by ``tangents``, but not their magnitude.
+        :param tolerance: tolerance of the algorithm (consult OCC documentation)
+                Used to check that the specified points are not too close to each
+            other, and that tangent vectors are not too short. (In either case
+            interpolation may fail.)
+                Set to None to use the default tolerance.
```

Comment:
Could you please fix the leading whitespace? If you check the details of the Read The Docs test, you'll see a build of the HTML docs and some bad formatting resulting from this whitespace. https://cadquery--636.org.readthedocs.build/en/636/classreference.html#cadquery.Workplane.spline

Reply:
`sphinx-build` is currently failing on my machine. I'll need to figure out how to test this.

Reply:
A added a commit with an attempted fix.

Reply:
Read The Docs build LGTM.